### PR TITLE
Bump Iceberg to 0.13.2

### PR DESCRIPTION
**Pull request by nastra**
Iceberg 0.13.2 is about to be released and this PR makes sure that existing tests work with the new version

### CODE REVIEW

File: buildSrc/src/main/kotlin/nessie-conventions.gradle.kts [hunk 1]
```diff
@@ -160,6 +160,7 @@ fun Project.configureJava() {
     if (System.getProperty("withMavenLocal").toBoolean()) {
       mavenLocal()
     }
+    maven { setUrl("https://repository.apache.org/content/repositories/orgapacheiceberg-1088/") }
```
Comment (snazy) [inline buildSrc/src/main/kotlin/nessie-conventions.gradle.kts:163]: Move that up before `mavenCentral` and add a `content` + `include` rule to only include Iceberg.

Comment (snazy) [reply buildSrc/src/main/kotlin/nessie-conventions.gradle.kts:163]: And give it a `name`

Comment (nastra) [reply buildSrc/src/main/kotlin/nessie-conventions.gradle.kts:163]: this will be removed once 0.13.2 has been released

**Comment by nastra** [comment]
superseded by https://github.com/projectnessie/nessie/pull/4477 thus not needed anymore

**Pull request closed without merging**